README.md

```diff
@@ -61,6 +61,7 @@ Renderer options:
   -q, --qzone       Quiet zone size                                     [number]
   -l, --lightcolor  Light RGBA hex color
   -d, --darkcolor   Dark RGBA hex color
+  --small  Output smaller QR code to terminal                          [boolean]
```

Comment:
Why not reuse `--scale`?

Reply:
No problem. What value would you suggest? Anything below 1 should use the smaller version?